src/routes/balances/balances.service.ts

```diff
@@ -161,7 +161,7 @@ export class BalancesService {
 
     return <Balances>{
       fiatTotal: getNumberString(fiatTotal),
-      items: orderBy(balances, 'fiatBalance', 'desc'),
+      items: orderBy(balances, (b) => Number(b.fiatBalance), 'desc'),
```

Comment:
Exactly, that's the reason if I recall correctly. When the price provider FF is fully applied (and therefore we have just one source for the prices), I'd say we can revisit these domain entities. From my perspective, it makes more sense to use numbers in the domain layer (and it would make the calculations easier also) and just map to `string` in the routes layer to satisfy the clients' requirements.